assets/vue/components/dashboard/TheHistoAppDashboard.vue

```diff
@@ -0,0 +1,245 @@
+        this.sharedState.territories.push(optionItem)
+      }
+    },
+
+    handleInitKPI (requestResponse: any) {
+      if (requestResponse === 'error') {
+        this.isErrorInit = true
+      } else {
+        this.isLoadingInit = false
+        this.isLoadingRefresh = false
+        this.processResponseInit(requestResponse)
+        if (this.sharedState.user.isAdmin || this.sharedState.user.isResponsableTerritoire) {
+          this.countTablesToLoad++
+          requests.initAffectationPartner(this.handleAffectationPartner)
+        }
+        if (this.sharedState.user.isResponsableTerritoire) {
+          this.countTablesToLoad++
+          requests.initSignalementsNoSuivi(this.handleSignalementsNoSuivi)
+        }
+        if (this.sharedState.user.isAdmin) {
+          this.countTablesToLoad++
+          requests.initSignalementsPerTerritoire(this.handleSignalementsPerTerritoire)
+          this.countTablesToLoad++
+          requests.initEsaboraEvents(this.handleEsaboraEvents)
+        }
+      }
+    },
+    processResponseInit (requestResponse: any) {
+      this.sharedState.signalements.count = requestResponse.data.countSignalement.active
+      this.sharedState.signalements.percent = requestResponse.data.countSignalement.percentage.active
+      this.sharedState.closedSignalements.count = requestResponse.data.countSignalement.closed
+      this.sharedState.closedSignalements.percent = requestResponse.data.countSignalement.percentage.closed
+      this.sharedState.newSignalements.count = requestResponse.data.countSignalement.new
+      this.sharedState.newSignalements.percent = requestResponse.data.countSignalement.percentage.new
+      this.sharedState.refusedSignalements.count = requestResponse.data.countSignalement.refused
+      this.sharedState.refusedSignalements.percent = requestResponse.data.countSignalement.percentage.refused
+
+      this.sharedState.suivis.countMoyen = requestResponse.data.countSuivi.average
+      this.sharedState.suivis.countByPartner = requestResponse.data.countSuivi.partner
+      this.sharedState.suivis.countByUsager = requestResponse.data.countSuivi.usager
+
+      this.sharedState.users.countActive = requestResponse.data.countUser.active
+      this.sharedState.users.percentActive = requestResponse.data.countUser.percentage.active
+      this.sharedState.users.countNotActive = requestResponse.data.countUser.inactive
+      this.sharedState.users.percentNotActive = requestResponse.data.countUser.percentage.inactive
+
+      const dataWidget = requestResponse.data.widgetCards
+      if (dataWidget.cardNouveauxSignalements !== undefined) {
+        this.sharedState.newSignalements.count = dataWidget.cardNouveauxSignalements.count
+        this.sharedState.newSignalements.link = dataWidget.cardNouveauxSignalements.link
+      }
+      if (dataWidget.cardAffectation !== undefined) {
+        this.sharedState.newAffectations.link = dataWidget.cardAffectation.link
+      }
```

Comment:
Je pense qu'on peut virer tous les if en utilisant l'opérateur ?. comme en php : 

 `       this.sharedState.newAffectations.link = dataWidget.cardAffectation?.link`